experiment/builder_runner.py

```diff
@@ -110,7 +110,11 @@ def _libfuzzer_args(self) -> list[str]:
         f'-max_total_time={self.run_timeout}',
         # Without this flag, libFuzzer only consider short inputs in short
         # experiments, which lowers the coverage for quick performance tests.
-        '-len_control=0'
+        '-len_control=0',
+        # Timeout per testcase.
+        '-timeout=30',
+        # Truncate testcase.
+        '-max_len=4096',
```

Comment:
This is a bit dangerous and can cause real bugs to be missed. Was there any reason in particular for this one?

Reply:
Ah, I was going to consult you about whether this is a good idea but forgot to comment on it.
I was worried about the same thing (missing bugs) but was wondering if this can help reduce OOM (e.g., many memory allocations depend on `data` or use `size`).

Let me remove it, then.